test/e2e/network.go

```diff
@@ -105,7 +105,7 @@ func bootstrapE2ENetwork(logFilePrefix string, virtualChainId primitives.Virtual
 				experimentalExternalProcessorPluginPath,
 			)
 
-		if deployBlocksFile {
+		if deployBlocksFile && i != METRICS_NODE_INDEX{
```

Comment:
add a comment here to say that we require the metrics node to sync his blocks from his peers?